data/db-config.js

```diff
@@ -0,0 +1,5 @@
+const knex = require('knex');
+
+const knexConfig = require('../knexfile.js');
```

Comment:
You can use an environment variable to make it easier for deploying to production